src/components/containers/modal/Modal.js

```diff
@@ -130,7 +130,11 @@ class Modal extends React.Component {
           show={show}
           transition={animation ? Fade : undefined}
         >
-          <ModalDialog size={size} aria-labelledby={this.state.ariaId}>
+          <ModalDialog
+            size={size}
+            aria-labelledby={this.state.ariaId}
+            className="modal-dialog"
```

Comment:
I like this. I only question if this is too rigid or if we need to add the flexibility of changing this to `className={this.props.className}`.

Thoughts? @sbartlett @jrios @Darrken

Reply:
Eh, I don't know that it needs to be that flexible. If we were to do anything, we could add it this way:

```
const modalDialogClasses = `modal-dialog ${this.props.className}`
```

I'd say it's pretty unnecessary at this point, though.